images/Utils.ps1

```diff
@@ -258,7 +258,7 @@ $Images = @(
     DockerImage -Name "dnsutils" -ImageBase "busybox" -Versions "1.1"
     DockerImage -Name "echoserver" -ImageBase "busybox" -Versions "2.2"
     DockerImage -Name "entrypoint-tester"
-    DockerImage -Name "etcd" -Versions "v3.3.10"
+    DockerImage -Name "etcd" -Versions "v3.3.10", "3.3.10"
```

Comment:
does this image exist in both v* and just * versions. weird...

Reply:
it has to be in both versions. For master tests, it has been renamed to "3.3.10" but for 1.14, the change doesn't exist. Thus we have to go ahead maintaining both versions until either we stop testing 1.14 or the changes is cherry-picked to 1.14 branch ( which I doubt will ever happen, as it's not a bug fix )